addons/medical_ai/XEH_postInit.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+
+["ace_settingsInitialized", {
+    // Only run for AI that does not have to deal with advanced medical
+    if (EGVAR(medical,enableFor) == 1 || {hasInterface && {EGVAR(medical,level) == 2}}) exitWith {};
+
+    ["ace_firedNonPlayer", {
+        _unit setVariable [QGVAR(lastFired), CBA_missionTime];
+    }] call CBA_fnc_addEventHandler;
+
+    if (hasInterface) then {
+        ["ace_unconscious", {
```

Comment:
This is a global event. Some kind of locality check or duplication check will be necessary in this event handler. No need to add to the healQueue if the unit is already in that queue